index.js

```diff
@@ -23,6 +23,8 @@ class ChatMessage {
       this.json = { text: message }
     } else if (typeof message === 'object' && !Array.isArray(message)) {
       this.json = message
+    } else if (typeof message === 'object') {
```

Comment:
this is adding a case when message is *not* an array
this isn't what the pr message is saying

what do you actually want to implement ?

Reply:
Line 26 checks that the message is an array, since the else if right before it checks that the message isn't an array. I guess I could add `&& Array.isArray(message)` to make it more obvious but that wouldn't really change any behavior.

Reply:
right I missed the !, all good